cloudinary/utils.py

```diff
@@ -1264,6 +1264,56 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def compute_hex_hash(s):
+    """
+    Compute hash and convert the result to HEX string
+
+    :param s: string to process
+
+    :return: HEX string
+    """
+    return hashlib.sha1(to_bytes(s)).hexdigest()
+
+
+def verify_api_response_signature(public_id, version, signature):
+    """
+    Validates API response signature against Cloudinary configuration.
+
+    :param public_id: Public ID of resource
+    :param version: Version of resource
+    :param signature: Response signature
+
+    :return: Boolean result of the validation
+    """
+    parameters_to_sign = {'public_id': public_id,
+                          'version': version}
+    signed_parameters = api_sign_request(parameters_to_sign, cloudinary.config().api_secret)
```

Comment:
Please verify that api_secret is not empty and throw exception if it is. Add unit test.
:+1: for reusing `api_sign_request`